### PR TITLE
v1.4.0_SSR-2030: Add ja and en release notes

### PR DESCRIPTION
**Pull request by TomokoMiyake**
## Checklist

- [x] Read CONTRIBUTING.md([for v1](https://github.com/kintone-labs/kintone-ui-component/master/CONTRIBUTING.md), [for v0](https://github.com/kintone-labs/kintone-ui-component/v0_dev/CONTRIBUTING.md))
- [x] Updated documentation if it is required.
- [x] Added tests if it is required.
- [x] Passed lint and test command.

### CODE REVIEW

File: docs/document/docs/releases/release-notes-v1.4.0.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+id: release-notes-v1.4.0
+title: v1.4.0 Release Notes
+sidebar_label: v1.4.0 Release Notes
+---
+
+## Overview
+
+Here are the [kintone UI Component v1.4.0](https://github.com/kintone-labs/kintone-ui-component/releases/tag/v1.4.0) Release Notes.<br>
+New components, features, maintenances, document updates, and security updates have been made.
```
Comment (huuchi1778) [inline docs/document/docs/releases/release-notes-v1.4.0.md:10]:
```suggestion
New components, features, maintenance, document updates, and security updates have been made.
```

File: docs/document/docs/releases/release-notes-v1.4.0.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+> Note that in this version, there is backward compatible breaking change with coding way and components' tags and class names for resolving version conflicts issue.
+> For more details, please check the [Version conflicts issue and solution](../guides/version-conflicts-issue-solution) and new [Quick Start](../getting-started/quick-start) article.
```
Comment (huuchi1778) [inline docs/document/docs/releases/release-notes-v1.4.0.md:13]:
```suggestion
> For more details, please check the [Version conflicts issue and solution](../guides/version-conflicts-issue-solution) and the new [Quick Start](../getting-started/quick-start) article.
```

File: docs/document/docs/releases/release-notes-v1.4.0.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+### Maintenance
+- Resolved version conflicts issue.
+- Exported type definition.
+- Fixed style of Mobile Date/Time series.
+
+### Security Updates
+- Updated dependent libraries: nth-check, core-js, webpack, webpack-cli, @storybook/builder-webpack5, @storybook/manager-webpack5, @storybook/web-components, @storybook/addon-actions, @storybook/addon-controls, @storybook/addon-viewport, @storybook/addon-a11y, karma-coverage, @babel/preset-env, @babel/core, babel-loader, ts-loader, @open-wc/testing, karma, prettier, @cybozu/eslint-config, lit, @types/uuid, typescript.
+
+### Document
+- Added article to explain version conflicts issue and solution and updated quick start description and component sample code and README.
```
Comment (huuchi1778) [inline docs/document/docs/releases/release-notes-v1.4.0.md:32]:
```suggestion
- Added an article to explain the version conflicts issue and solution, updated quick start description, component sample code and README.
```

File: docs/document/docs/releases/release-notes-v1.4.0.md (hunk 1 above)
Comment (TomokoMiyake) [reply docs/document/docs/releases/release-notes-v1.4.0.md:10]: It's good! I will update it.

**Pull request merged**